core/src/server/rpc_module.rs

```diff
@@ -416,36 +374,36 @@ impl Methods {
 
 	/// Execute a callback.
 	async fn inner_call(&self, req: Request<'_>) -> RawRpcResponse {
-		let (tx_sink, mut rx_sink) = mpsc::unbounded();
+		let (tx_sink, mut rx_sink) = mpsc::channel(u32::MAX as usize / 2);
```

Comment:
This is essentially an unbounded channel that's handed to the subscription handler; won't this defeat the purpose of bounded channels or am I missing something?

Reply:
this is just used for tests :)

Reply:
but perhaps we could use this for testing some backpressure stuff without a WebSocket server.